src/js/actions/ActionsCreators.js

```diff
@@ -280,8 +279,11 @@ export function setPayout(payload) {
     if (typeof payout === 'undefined') {
       payout = getState().getIn(['storage', 'payout']);
     }
+    else if (isNaN(payout*1) || !Number.isInteger(payout*1) || parseInt(payout) < 0 || parseInt(payout) > 100) {
```

Comment:
Oh, totally, guess I was blind

it should be 

let payout = payload && (+payload.payout);
at 
https://github.com/negar-binary/japanui/blob/6051a4c5a6e9c64acce64b1dd366a0bf02a75311/src/js/actions/ActionsCreators.js#L275

By the way, why are we checking whether `typeof` is undefined or not, instead of value ?

generally my suggestion is to use `+` rather than `parseInt`, and reduce unnecessary usage of isNaN which convolute the logic